db/db_impl/db_impl.cc

```diff
@@ -1142,6 +1142,7 @@ std::string DescribeVersionEdit(const VersionEdit& e, ColumnFamilyData* cfd) {
 
 Status DBImpl::ApplyReplicationLogRecord(ReplicationLogRecord record,
                                          std::string replication_sequence,
+                                         CFOptionsFactory cf_options_factory,
```

Comment:
I guess when time partitioning is enabled, we will have different cf options for non-default cfs so we need this? Or is this for some other cases?

Reply:
nvm, saw your msg in slack.